src/frontend/src/app/AppAuthenticated.tsx

```diff
@@ -49,11 +47,9 @@ const AppAuthenticated: React.FC<AppAuthenticatedProps> = ({ userId }) => {
 
   return userSettingsData.slackId ? (
     <AppContextUser>
-      <NavTopBar handleDrawerOpen={handleDrawerOpen} open={drawerOpen} />
       <Box display={'flex'}>
-        <Sidebar handleDrawerClose={handleDrawerClose} open={drawerOpen} />
+        <Sidebar handleDrawerClose={handleDrawerClose} handleDrawerOpen={handleDrawerOpen} open={drawerOpen} />
```

Comment:
is there any reason the drawerOpen useState can't just be inside our Sidebar component?